### PR TITLE
Dbz 1988 skip lsn timestamp query using oracle cdc jobs

### PR DESCRIPTION
**Pull request by jantpedraza**

### CODE REVIEW

**Comment by gunnarmorling** [comment]
@jantpedraza, could you please run `mvn process:sources` to apply the formatter? Could you also rework the commits to use an email address associated to your GitHub account? Thanks!

File: documentation/modules/ROOT/pages/connectors/sqlserver.adoc [hunk 1]
```diff
@@ -1514,6 +1514,12 @@ twice - once in initial snapshot and once in streaming phase. Nonetheless, that
 data mirroring.
 For `read_uncommitted` there are no data consistency guarantees at all (some data might be lost or corrupted).
 
+|[[connector-property-source-timestamp-mode]]<<connector-property-source-timestamp-mode>>, `source.timestamp.mode`
+|_commit_
+|String representing the criteria of the attached timestamp within the source record (ts_ms).
+`commit` will set the source timestamp to the instant where the record was committed in the database
+`processing` will set the source timestamp to the instant where the record was processed by Debezium.
```
Comment (gunnarmorling) [inline documentation/modules/ROOT/pages/connectors/sqlserver.adoc:1521]: Could you add a sentence describing when/why one would use `processing`?

**Comment by gunnarmorling** [comment]
Hey @jantpedraza, if you have a chance to address the remarks above, we could include this into next week's release of 1.2 Beta2 (planned for Tuesday). Thanks!

File: documentation/modules/ROOT/pages/connectors/sqlserver.adoc (hunk 1 above)
Comment (jantpedraza) [reply documentation/modules/ROOT/pages/connectors/sqlserver.adoc:1521]: Sure!

**Comment by jantpedraza** [comment]
Yeah @gunnarmorling, I am now applying your suggestions :)

**Comment by gunnarmorling** [comment]
@jantpedraza, please see [here](https://help.github.com/en/github/setting-up-and-managing-your-github-profile/why-are-my-contributions-not-showing-up-on-my-profile) for the details. In a nutshell, the commit must be authored using an e-mail address associated to your GH account. Just run

    git commit --amend --author "Your name <youremailaddress>"
    git push -f origin DBZ-1988-skip-lsn-timestamp-query-using-oracle-cdc-jobs

Thanks!

**Comment by jantpedraza** [comment]
Morning @gunnarmorling, all yours! these changes applied and commit tag with author :)

Thanks a lot for your help and patience :)

**Comment by gunnarmorling** [comment]
Thanks, @jantpedraza. Closing this in favour of #1508, where I most notably added an actual integration test. This revealed the option actually was not applied (so it was still using COMMIT no matter what), since the option was read from the JDBC config map to which it isn't propagated, though.

**Pull request closed without merging**